src/hyper-log-catcher/HyperLogger.res

```diff
@@ -86,6 +86,9 @@ type eventName =
   | EXTERNAL_TAX_CALCULATION
   | POST_SESSION_TOKENS_CALL
   | POST_SESSION_TOKENS_CALL_INIT
+  | PAZE_SDK_FLOW
+  | SAMSUNG_PAY_SCRIPT
+  | SAMSUNG_PAY_FLOW
```

Comment:
```suggestion
  | SAMSUNG_PAY
```

Reply:
change the  event name to SAMSUNG_PAY where ever used.

Reply:
Updated